apps/landing/components/layouts/Home/Footer/UpperPart.tsx

```diff
@@ -59,28 +62,45 @@ const UpperPart = () => {
 					!
 				</Text>
 
-				<Stack
-					paddingVertical={8}
-					paddingHorizontal={4}
-					borderWidth={1}
-					borderRadius={16}
-					borderColor="#566674"
-					flexDirection="row"
-					width="100%"
-					maxWidth={600}
-				>
-					<Input
-						backgroundColor="transparent"
-						flexGrow={1}
-						placeholder="Enter your email"
-						borderWidth={0}
-					/>
-					<Button borderRadius={12} width={180}>
-						<Text fontWeight="600" fontSize={fontSize}>
-							Count me in
-						</Text>
-					</Button>
-				</Stack>
+				{isSignedUp ? (
+					<Text
+						maxWidth={420}
+						fontSize={fontSize}
+						lineHeight={fontSize * 1.5}
+						color="#566674"
+						textAlign="center"
+					>
+						Thanks for signing up. We&apos;ll keep you posted with Walless
+						latest updates and exclusive perks.
+					</Text>
+				) : (
+					<Stack
+						paddingVertical={8}
+						paddingHorizontal={4}
+						borderWidth={1}
+						borderRadius={16}
+						borderColor="#566674"
+						flexDirection="row"
+						width="100%"
+						maxWidth={600}
+					>
+						<Input
+							backgroundColor="transparent"
+							flexGrow={1}
+							placeholder="Enter your email"
+							borderWidth={0}
+						/>
+						<Button
+							borderRadius={12}
+							width={180}
```

Comment:
prefer sizing Button with padding over fixed width, if there is no specific reason.. rethink about this.